academic-requirements/src/components/InputPage.tsx

```diff
@@ -129,12 +141,15 @@ const InputPage = (props: {
     console.log("Deleted course: " + course);
   }
 
-  function importSchedule() {
-    //TODO check if the imported file is a valid format (jsonschema)
-    //TODO throw error if the input is not of correct format
-    throwError("Import file is not in a valid format");
-    /*TODO either update existing variables on screen
-   or bypass checking of those variables based on valid import*/
+  /*
+  This function calls the showUploader function 
+  it has the uploader popup appear where the user
+  can select and upload a chosen file.
+  Implementation of the Import Button and part of what it's suppose to do is in /ImportPopup/index.js
+  */
+  function setupUploader() {
```

Comment:
This will work but you can probably call `showUploader()` directly if this function doesn't need to do anything else in the future.